sdk/monitor/Azure.Monitor.Query/src/Models/LogsQueryResults.cs

```diff
@@ -20,11 +21,27 @@ public partial class LogsQueryResult
         [CodeGenMember("render")]
         private readonly JsonElement _visualization;
 
-        // TODO: Handle not found
         /// <summary>
-        /// Returns the primary result of the query.
+        /// Gets the single table result of the query.
         /// </summary>
-        public LogsQueryResultTable PrimaryTable => Tables.Single(t => t.Name == "PrimaryResult");
+        public LogsQueryResultTable Table
+        {
+            get
+            {
+                if (AllTables.Count != 1)
+                {
+                    throw new InvalidOperationException($"The result contains multiple tables, please use the {nameof(AllTables)} collection to access all tables.");
+                }
+
+                return AllTables[0];
+            }
+        }
+
+        /// <summary>
+        /// Gets the tables returned as the result of the query.
```

Comment:
Modify to more closely resemble the description of the `Table` property.

```suggestion
        /// Gets the multi-table result of the query.
```